DSL/Ruuter.private/DSL/POST/classifier/integration/jira/cloud/toggle-subscription.yml

```diff
@@ -0,0 +1,79 @@
+declaration:
+  call: declare
+  version: 0.1
+  description: "Description placeholder for 'TOGGLE-SUBSCRIPTION'"
+  method: post
+  accepts: json
+  returns: json
+  namespace: classifier
+  allowlist:
+    body:
+      - field: is_connect
+        type: boolean
+        description: "Body field 'isConnect'"
+
+extract_request_data:
+  assign:
+    is_connect: ${incoming.body.isConnect}
+  next: get_auth_header
+
+get_auth_header:
+  call: http.post
+  args:
+    url: "[#CLASSIFIER_DMAPPER]/hbs/classifier/get_auth_header"
+    headers:
+      type: json
+    body:
+      username: "[#JIRA_USERNAME]"
+      token: "[#JIRA_API_TOKEN]"
+  result: auth_header
+  next: check_integration_type
+
+check_integration_type:
+  switch:
+    - condition: ${is_connect === true}
+      next: subscribe_jira
+  next: unsubscribe_jira
+
+subscribe_jira:
+  call: http.put
+  args:
+    url: "[#JIRA_CLOUD_DOMAIN]/rest/webhooks/1.0/webhook/[#JIRA_WEBHOOK_ID]"
+    headers:
+      Authorization: ${auth_header.response.body.val}
+    body:
+      enabled: true
+  result: res
```

Comment:
Any reason we are naming the output as "res" here @kalsaraM, I noticed in the previous endpoints you had names which describes their function like "auth_header".

Reply:
it just a short name for reponse